src/sparql/update/remove-duplicate-synonyms.ru

```diff
@@ -0,0 +1,36 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+
+DELETE {
+ ?cls ?synonym ?value .
+    ?x a owl:Axiom ;
+      owl:annotatedSource ?cls ;
+      owl:annotatedProperty ?synonym ;
+      owl:annotatedTarget ?value ;
+      ?p ?z .
+}
+INSERT {
+ ?cls oboInOwl:hasExactSynonym ?value .
+    ?x a owl:Axiom ;
+      owl:annotatedSource ?cls ;
+      owl:annotatedProperty oboInOwl:hasExactSynonym ;
+      owl:annotatedTarget ?value ;
+      ?p ?z .
+}
+WHERE {
+  VALUES ?synonym {oboInOwl:hasExactSynonym oboInOwl:hasRelatedSynonym oboInOwl:hasNarrowSynonym oboInOwl:hasBroadSynonym} 
```

Comment:
I recommend changing this to

```
  VALUES ?synonym { oboInOwl:hasRelatedSynonym oboInOwl:hasNarrowSynonym oboInOwl:hasBroadSynonym} 
  VALUES ?synonym2 {oboInOwl:hasExactSynonym } 
```

This ensures that at least one of the two baldy scoped synonyms is exact before assuming they both are.

Reply:
done